data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SchedulingParameterConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SchedulingParameterConfiguration {
+
+    @JsonProperty("rate")
+    private String rate;
+
+    @JsonProperty("job_count")
+    private int jobCount;
+
+    @JsonProperty("start_time")
```

Comment:
We should validate that this is a valid timestamp, and let's default to `Instant.now()`

Reply:
Thanks @graytaylor0  for the review comments. The changes are incorporated.